packages/vincent-tools/vincent-tool-erc20-approval/src/lib/tool-helpers/get-pkp-info.ts

```diff
@@ -0,0 +1,41 @@
+import { parseAbi } from 'viem';
+import { createChronicleYellowstoneViemClient } from '.';
```

Comment:
Import directly from sibling file to avoid circular reference (index imports this file)